client/src/config/theme.ts

```diff
@@ -0,0 +1,20 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      light: "#f05545",
+      main: "#b71c1c",
+      dark: "#7f0000",
+      contrastText: "#fafafa",
+    },
```

Comment:
Nie ma jakiegoś enuma który może zastąpić stringa z kodem hexagonalnym danego koloru? 
Pytam tak z ciekawości.

Reply:
Niestety, byłoby to baardzo dużo enumów żeby pokryć każdy kolor, materialUI posiada jakieś predefiniowane, ale nie pokrywają się raczej z wybranymi - docsy: https://material-ui.com/customization/color/

Reply:
Ponadto, myślę że do wielu edytorów są extensions/pluginy które umożliwiają pokazanie rzeczywistego koloru obok tych zakodowanych (np taki do VS Code https://marketplace.visualstudio.com/items?itemName=kamikillerto.vscode-colorize)